libnethack/include/rtp.h

```diff
@@ -0,0 +1,42 @@
+#ifndef RTP_H
+#define RTP_H
+
+#include "extern.h"
+#include "hack.h"
+#include "monst.h"
+
+#define RTP_NAME_COUNT 16
+
+// These are added in order of when I got permission
```

Comment:
Nobody in the future will care. Just alphabetically sort these.

Reply:
Okay, just to note that also adds complexity if any more female RTP names are added because the macro below will need to be updated instead of just adding a new check.